src/Android/MainActivity.cs

```diff
@@ -30,6 +30,16 @@ namespace Bit.Droid
         ConfigurationChanges = ConfigChanges.ScreenSize | ConfigChanges.Orientation |
                                ConfigChanges.Keyboard | ConfigChanges.KeyboardHidden |
                                ConfigChanges.Navigation)]
+    [IntentFilter(
```

Comment:
Assuming the `IntentFilterAttribute` doesn't restrict by filter based on the intent actions exclusively (but that is how it's applying the filter when filtering instead; if types match then filter based on X)?

Reply:
Yeah, Microsoft's docs are useless here, figures: https://docs.microsoft.com/en-us/dotnet/api/android.app.intentfilterattribute?view=xamarin-android-sdk-9

Reply:
I'm assuming since the attribute target allows multiple, it would imply this acts as a filter for _when_ to apply the filter itself, vs. being part of the filter.

Reply:
@cscharf I've read this 20 times and I'm still not sure what you're asking me.  :)  The MS docs are also confusing me since it keeps redirecting everything back to Android 9...

Reply:
Yeah, sorry Matt, it was just that I wasn't sure that the values being passed in the string[] array as the first argument of the `[IntentFilter]` would filter/limit the call to only those actions exclusively, making it "not fire" for any other actions than what's listed in the `[IntentFilter]`, so basically asking if this filter would cause any actions OTHER than `new[] { Intent.ActionSend }` to be filtered out/rejected. I don't think so however, so I believe you're good. I think you can ignore my concern.

Reply:
Ah ok, I think I follow now, and your assumption is correct - the filters only apply to this intent action.  Other actions can be declared with their own set of filters, even if they include the same mime types.